.github/workflows/upstream_testing.yml

```
@@ -1,7 +1,11 @@
 ---
 name: "Nautobot Upstream Monitor"
 
+<<<<<<< HEAD
```

actionlint (1.7.11)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)